apis/cosmos-reducers.js

```diff
@@ -584,10 +584,7 @@ export function transactionReducer(transaction) {
         return coinReducer(fee, coinLookup)
       })
     }
-    const {
-      claimMessages,
-      otherMessages,
-    } = transaction.tx.body.messages.reduce(
+    const { claimMessages, otherMessages } = transaction.tx.value.msg.reduce(
```

Comment:
no

Reply:
```suggestion
    const { claimMessages, otherMessages } = transaction.tx.body.messages.reduce(
```

Reply:
I tested with both e-Money and Cosmos Hub :shrug: